Makefile

```
@@ -45,8 +45,11 @@ layers: docker-images
 			echo "### Exporting $${dir} PHP$${php_version}"; \
 			echo "###"; \
 			cd ${PWD} ; rm -rf export/tmp/${layer} || true ; cd export/tmp ; \
-			docker run --entrypoint "tar" bref/$${dir}-php-$${php_version} -ch -C /opt . | tar -x ; \
-			zip --quiet -X --recurse-paths ../`echo "$${dir}-php-$${php_version}" | sed -e "s/layers\//layer-/g"`.zip . ; \
+			CID=$$(docker create --entrypoint=/bin/bash bref/$${dir}-php-$${php_version}) ; \
```

Indeed I've checked too, this is pure detail but as it's hacky, I suggest to set an entrypoint that explain we should expect bash here, maybe by reusing the `scratch` term?
```suggestion
			CID=$$(docker create --entrypoint=scratch bref/$${dir}-php-$${php_version}) ; \
```

---

Yes, This make sense. 

Thank you